build.sbt

```diff
@@ -57,7 +67,8 @@ def configCommonTestSettings(scopes: String) = Seq(
   libraryDependencies ++= Seq (
     "com.novocode" % "junit-interface" % "0.11" % scopes,
     "org.assertj" % "assertj-core" % "3.4.1" % scopes,
-    "com.icegreen" % "greenmail" % "1.5.0" % scopes
+    "com.icegreen" % "greenmail" % "1.5.0" % scopes,
+    "org.mockito" % "mockito-core" % "1.10.19" % scopes
```

Comment:
I actually dislike the usage of mockito and similar frameworks. If you don't use a mocking framework then you see faster API design problems and refactor until it is easy testable.